src/runtime/env.rs

```diff
@@ -440,6 +472,34 @@ mod test {
         );
     }
 
+    #[tokio::test]
+    async fn test_migration_to_latest_version() {
+        {
+            let _test_env_guard = TestEnv::reset().expect("Should lock TestEnv");
+            let profile_before = json!({
+                "settings": {}
+            });
+
+            // setup storage for migration
+            set_profile_and_schema_version(&profile_before, 1);
+
+            // migrate storage
+            TestEnv::migrate_storage_schema()
+                .await
+                .expect("Should migrate");
+
+            let storage = STORAGE.read().expect("Should lock");
+
+            assert_eq!(
+                &SCHEMA_VERSION.to_string(),
```

Comment:
I believe this should now be 7?
I would suggest to do as the other tests - name the test `v6 to v7` and use hardcoded value instead of `SCHEMA_VERSION`. This will eliminate the need for this test to be update on a new schema.

Reply:
This test is specifically like this so you wouldn't need to change it and it would test that the migrations really update to the latest defined version, as this runs the whole migration process not a single migration method.

Reply:
No, you shouldn't need to change anything in this test when adding a new version if you do everything correctly.
This will test and fail if you:
 - create a new migration and bump the schema version in the migration, but forget to update the const `SCHEMA_VERSION`
 - you bump the `SCHEMA_VERSION` but forget to create a migration for the new schema version
 - you create the migration and bump the `SCHEMA_VERSION`, but in your migration you've forgot to update the schema version.